frontend/public/components/utils/resource-log.jsx

```diff
@@ -23,12 +25,16 @@ export const LOG_SOURCE_WAITING = 'waiting';
 const streamStatusMessages = {
   [STREAM_EOF]: 'Log stream ended.',
   [STREAM_LOADING]: 'Loading log...',
-  [STREAM_PAUSED]: 'Log stream paused.',
-  [STREAM_ACTIVE]: 'Log streaming...'
+  [STREAM_PAUSED]: 'Container:',
+  [STREAM_ACTIVE]: 'Container:'
```

Comment:
If we're going to make this change, we should change all of these messages and get rid of this object. I'd prefer to look at this separately if you don't mind @beanh66

Reply:
It depends on if we still want to dynamically show there
```
[STREAM_EOF]: 'Log stream ended.',
[STREAM_LOADING]: 'Loading log...',
```

Reply:
If we want "Container:" we should always show that, but then we need to find another place for these messages.

I'd rather address in a follow on since it's an unrelated change and this PR is almost ready.

Reply:
Ok, reverted.